front/pages/w/[wId]/assistant/new.tsx

```diff
@@ -1,57 +1,72 @@
 import {
```

Comment:
There's a little bit of scroll on the page, even though everything fits in the page. Can we fix that ?